operator/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# syntax=docker/dockerfile:1.1-experimental
+
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
```suggestion
# Copyright 2021 The Kubernetes Authors.
```

Reply:
I made the change for this PR, but just FYI, I've removed this file in a subsequent PR #4126.